.circleci/config.yml

```diff
@@ -323,46 +294,20 @@ workflows:
           context:
             - dockerhub
             - sonarcloud
+            - oicr-slack
       - integration_test:
-          name: integration_test_1
-          integration_test_name: group1
-          filters:
-            tags:
-              only: /.*/
-          requires:
-            - lint_license_unit_test_coverage
-          context:
-            - dockerhub
-      - integration_test:
-          name: integration_test_2
-          integration_test_name: group2
-          filters:
-            tags:
-              only: /.*/
-          requires:
-            - lint_license_unit_test_coverage
-          context:
-            - dockerhub
-      - integration_test:
-          name: integration_test_3
-          integration_test_name: group3
-          filters:
-            tags:
-              only: /.*/
-          requires:
-            - lint_license_unit_test_coverage
-          context:
-            - dockerhub
-      - integration_test:
-          name: integration_test_4
-          integration_test_name: immutableDatabaseTests
+          matrix:
+            parameters: 
+              integration_test_name: ["group1", "group2", "group3", "immutableDatabaseTests"]
```

Comment:
This uses `matrix` so we don't have to define a job for each integration test. One consequence of this is the the integration test names are no long `integration_test_1`, `integration_test_2`, etc. They are now `integration_test_<test-name>` like `integration_test_group1`, and `integration_test_immutableDatabaseTests`